GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -15,6 +15,81 @@
  */
 
 #import <Foundation/Foundation.h>
+#import <TargetConditionals.h>
 
+#if __has_include(<UIKit/UIKit.h>)
+#import <UIKit/UIKit.h>
+#elif __has_include(<AppKit/AppKit.h>)
+#import <AppKit/AppKit.h>
+#endif
+
+@class GIDVerifiableAccountDetail;
+@class GIDVerifiedAccountDetailResult;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// This class is used to verify a user's Googleaccount details.
 @interface GIDVerifyAccountDetail : NSObject
+
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+
+/// Starts an interactive verify flow on iOS.
+///
+/// The completion will be called at the end of this process.  Any saved verify state will be
+/// replaced by the result of this flow.
+///
+/// @param accountDetails A list of verifiable account details.
+/// @param presentingViewController The view controller used to present `SFSafariViewController` on
+///     iOS 9 and 10 and to supply `presentationContextProvider` for `ASWebAuthenticationSession` on
+///     iOS 13+.
+/// @param completion The optional block that is called on completion.  This block will
+///     be called asynchronously on the main queue.
+- (void)verifyAccountDetails:(NSArray<GIDVerifiableAccountDetail *> *)accountDetails 
+         presentingViewController:(UIViewController *)presentingViewController 
+                       completion:(nullable void (^)(GIDVerifiedAccountDetailResult *_Nullable verifyResult, 
+                                                   NSError *_Nullable error))completion;
+
+/// Starts an interactive verify flow on iOS using the provided hint.
+///
+/// The completion will be called at the end of this process.  Any saved verify state will be
+/// replaced by the result of this flow.
+///
+/// @param accountDetails A list of verifiable account details.
+/// @param presentingViewController The view controller used to present `SFSafariViewController` on
+///     iOS 9 and 10 and to supply `presentationContextProvider` for `ASWebAuthenticationSession` on
+///     iOS 13+.
+/// @param hint An optional hint for the authorization server, for example the user's ID or email
+///     address, to be prefilled if possible.
+/// @param completion The optional block that is called on completion.  This block will
```

Comment:
See above comment for a suggestion on making this simpler.

Reply:
Done.